client/src/App.js

```diff
@@ -29,6 +31,22 @@ class App extends Component {
         this.refreshList();
     };
 
+    handleOnChange = async v => {
+        var searchText = v.toLowerCase();
+        const messages= await ApiClient.allMessage();
+
+        if (searchText !== ""){
+            var filteredMessages = messages.filter(message => message.license_plate.toLowerCase().includes(searchText));
```

Comment:
Works, but would be more powerful by doing it backend side. And probably even MySQL side. Also helps if you start having multiple frontends